aemedge/blocks/product-tabs/product-tabs.js

```diff
@@ -0,0 +1,154 @@
+/*
+ * Product Tabs block
+ * - Parses authored rows: Label | URL (anchor)
+ * - Supports extensionless URLs and absolute links; uses anchor.pathname
+ * - Auto-generates tabs using a canonical order if no rows are authored
+ * - Renders a crawlable nav with real links and active-state highlighting
+ */
+
+import { toClassName } from '../../scripts/aem.js';
+
+const CANONICAL_ORDER = ['overview', 'quotes', 'settlements', 'volume', 'specs', 'margins', 'calendar'];
+const CANONICAL_LABELS = {
```

Comment:
Please use i18n function for labels as it's used in:

`  const [allLabel, inProgressLabel, completedLabel, viewLabel] = await Promise.all([
    i18n('All'),
    i18n('In Progress'),
    i18n('Completed'),
    i18n('View:'),
  ]);`